tests/auth/test_controller.py

```diff
@@ -47,7 +47,7 @@ def test_throw_400_if_user_name_and_email_is_none(self):
                               content_type='application/json')
         data = json.loads(rv.data)
         assert rv.status_code == 400
-        assert data['error_code'] == 'INVALID_INPUT'
+        assert data['message'] == 'User name or email both can not be empty'
```

Comment:
The message is not clear here.. why has it "both" word? Probably it should be:
Username and email can not be empty.